app/config/campaigns.yml

```diff
@@ -74,3 +74,16 @@ address_type:
   url_key: addrpre
   active: true
   param_only: false
+
+address_provision_options:
+  description: Test adding provisional address type options before choosing a donor type. (Adds "e-mail only" option)
+  reference: "https://phabricator.wikimedia.org/T260023"
+  start: "2020-10-01"
+  end: "2020-12-31"
+  buckets:
+    - "provisional_address_options"
+    - "old_address_type_options"
+  default_bucket: "provisional_address_options"
+  url_key: provadd
+  active: true
+  param_only: false
```

Comment:
This should be `true` because we want to test this specifically with the banners in desktop test07. All other banner tests should get the "old style" address chooser.